modules/manage/pages/iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc

```diff
@@ -1,7 +1,6 @@
 = Query Iceberg Topics using Snowflake and Open Catalog
 :description: Add Redpanda topics as Iceberg tables that you can query in Snowflake using an Open Catalog integration.
 :page-categories: Iceberg, Tiered Storage, Management, High Availability, Data Replication, Integration
-:page-beta: true
```

Comment:
@kbatuigas some of the external links on this page need `^` to open in a new tab.

Reply:
Updated